trackma/ui/cli.py

```diff
@@ -945,6 +981,48 @@ def _make_list(self, showlist):
         print('%d results' % len(showlist))
         print()
 
+    def _make_list_json(self, showlist):
+        """
+        helper function for printing a json formatted show list
+        """
+        altnames = self.engine.altnames()
+
+        episode_str_current = ""
+        episode_str_last = ""
+        # list shows
+        for index, show in showlist:
+            if self.engine.mediainfo['has_progress']:
+                episode_str_current = "{0}".format(show['my_progress'] or '0')
+                episode_str_last = "{0}".format(show['total'] or '?')
+
+            # get title (and alt. title) and if need be, truncate it
+            title_str = show['title']
+            if altnames.get(show['id']):
+                title_str += " [{}]".format(altnames.get(show['id']))
```

Comment:
Better use an `altname` field than modify the `title`.

Reply:
I kept the code from _make_list function.
I think just having: `title_str = show['title']` should be enough; otherwise I am not sure of a better implementation using altnames.

Reply:
If you don't think the altname is useful (for you), you can just skip it). Either way, you should not alter the `title` here.

Reply:
Resolved. Simply `title_str = show['title']` and pass that to json.